fabric-convention-tags-v1/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/BlockTagGenerator.java

```diff
@@ -112,4 +114,59 @@ private void generateGlassTags() {
 				.add(Blocks.WHITE_STAINED_GLASS_PANE)
 				.add(Blocks.YELLOW_STAINED_GLASS_PANE);
 	}
+
+	private void generateJobSiteBlocksTag() {
+		getOrCreateTagBuilder(ConventionalBlockTags.VILLAGER_JOB_SITES)
+				.add(Blocks.BARREL)
+				.add(Blocks.BLAST_FURNACE)
+				.add(Blocks.BREWING_STAND)
+				.add(Blocks.CARTOGRAPHY_TABLE)
+				.add(Blocks.CAULDRON)
+				.add(Blocks.COMPOSTER)
+				.add(Blocks.FLETCHING_TABLE)
+				.add(Blocks.GRINDSTONE)
+				.add(Blocks.LECTERN)
+				.add(Blocks.LOOM)
+				.add(Blocks.SMITHING_TABLE)
+				.add(Blocks.SMOKER)
+				.add(Blocks.STONECUTTER);
+	}
+
+	private void generateSandstoneBlocksTag() {
+		getOrCreateTagBuilder(ConventionalBlockTags.YELLOW_SANDSTONE_BLOCKS)
+				.add(Blocks.SANDSTONE)
+				.add(Blocks.CUT_SANDSTONE)
+				.add(Blocks.CHISELED_SANDSTONE)
+				.add(Blocks.SMOOTH_SANDSTONE);
+		getOrCreateTagBuilder(ConventionalBlockTags.RED_SANDSTONE_BLOCKS)
+				.add(Blocks.RED_SANDSTONE)
+				.add(Blocks.CUT_RED_SANDSTONE)
+				.add(Blocks.CHISELED_RED_SANDSTONE)
+				.add(Blocks.SMOOTH_RED_SANDSTONE);
+		getOrCreateTagBuilder(ConventionalBlockTags.SANDSTONE_BLOCKS)
+				.addOptionalTag(ConventionalBlockTags.YELLOW_SANDSTONE_BLOCKS)
```

Comment:
Why optional btw, just curious?

Reply:
I'm not 100% sure, I used optional because thats what everything else uses. I can't see a reason, but I assumed there was one.